.github/workflows/unit-test.yml

```diff
@@ -72,9 +72,15 @@ jobs:
                     - '8.1'
                     - '8.2'
                 multisite: [false, true]
+                wordpress: [''] # Latest WordPress version.
+                include:
+                    # Test with the previous WP version.
+                    - php: '7.4'
+                      wordpress: '6.0'
 
         env:
             WP_ENV_PHP_VERSION: ${{ matrix.php }}
+            WP_ENV_CORE: WordPress/WordPress${{ matrix.wordpress != '' && format( '#{0}-branch', matrix.wordpress ) || '' }}
```

Comment:
I've tested this PR, and I don't think `-branch` needs to be appended to the name of the WP version.
When I log in to the docker container, I get `6.0.4-alpha-55372` as the WP version in `src/wp-includes/version.php`, but it should be `6.0` instead.

Reply:
Not sure I agree 😅 In the example you're giving, I'd want us to use `6.0.3`. That's currently the latest version from the `6.0` series that has all the bugfixes that were deemed critical enough to warrant a point release (well, three of them); and I believe that the auto-updater would update most people's WordPress installation to `6.0.3`, if they had previously installed `6.0` (or `6.0.1` or `6.0.3`).

Using `-branch` is a cheap way of getting that latest version (that doesn't require us to do any REST API lookups). Now since we're fetching our WordPress from git, we're not getting `6.0.3` exactly, but the latest state of the `6.0` branch, which might have a few commits on top; and which has a weird `6.0.4-alpha-something` version tag, as you've found.

I still find that preferable over using `6.0`. Why? Imagine there is a bug in `6.0` that causes a problem (manifested by a unit test error) if the Gutenberg plugin is active. We can fix that bug, and it could make its way into a `6.0.x` point release, but it'd never get into `6.0`! This means the tests just would keep failing.

The optimal solution would likely be to actually compute the latest version from the `6.0` series (via something like [this](https://github.com/WordPress/gutenberg/pull/46983#discussion_r1107726200)). Personally, I'd be fine keeping the `-branch` approach for a first iteration, and do a separate PR to do the version number computation later. Would that be okay?

Reply:
I see your point, and I agree it's better to use the latest minor release (if available). 
But, I think using '6.0-branch' is less optimal than '6.0.3' because regular WordPress users don't usually update their websites to the latest "branch" version.
They update to the latest minor or major release using the [WordPress dashboard](https://wordpress.org/documentation/article/dashboard-updates-screen/).
From my point of view, it makes more sense to test Gutenberg with WP versions that most users can use and update to.